### PR TITLE
issue: 1075811 Fix csbuild errors

### PR DESCRIPTION
**Pull request by igor-ivanov**
Signed-off-by: Igor Ivanov <igor.ivanov.va@gmail.com>

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/1448/ for details (Mellanox internal link).

File: src/vma/util/vma_list.h [hunk 1]
```diff
@@ -251,12 +251,14 @@ class  vma_list_t
 	inline T* get_and_pop_front() {
 		T* list_front = front();
 		pop_front();
+		((list_node<T, offset> *)m_list.head.next)->obj_ptr = NULL;
```
Comment (liranoz12) [inline src/vma/util/vma_list.h]: Hi @igor-ivanov,
Please consider remove this line and add the following link to [erase()](https://github.com/Mellanox/libvma/blob/master/src/vma/util/vma_list.h#L270) method :
node_obj->obj_ptr = NULL;

File: src/vma/util/vma_list.h [hunk 2]
```diff
@@ -251,12 +251,14 @@ class  vma_list_t
...
 		return list_front;
 	}
 
 	inline T* get_and_pop_back() {
 		T* list_back = back();
 		pop_back();
+		((list_node<T, offset> *)m_list.head.prev)->obj_ptr = NULL;
```
Comment (liranoz12) [inline src/vma/util/vma_list.h]: Same as above

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/1449/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/1458/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/1465/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/1470/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/1471/ for details (Mellanox internal link).

**Comment by igor-ivanov** [comment]
@OphirMunk it is ready to merge

File: src/vma/sock/sockinfo_tcp.cpp [hunk 3]
```diff
@@ -2350,7 +2345,6 @@ int sockinfo_tcp::listen(int backlog)
 			m_p_rx_ring = rx_ring_iter->first;
 		}
 		si_tcp_logdbg("sock state = %d", get_tcp_state(&m_pcb));
-		success = true;
```
Comment (OphirMunk) [inline src/vma/sock/sockinfo_tcp.cpp]: @igor-ivanov why did you drop this assignment? 
```
success = true;
```
it seems like changing the flow logic

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/1473/ for details (Mellanox internal link).

**Comment by OphirMunk** [comment]
Merged #427

**Pull request closed without merging**